jwst/extract_1d/soss_extract/atoca_utils.py

```diff
@@ -10,7 +10,7 @@
 from scipy.sparse import find, diags, csr_matrix
 from scipy.sparse.linalg import spsolve
 from scipy.interpolate import interp1d, RectBivariateSpline, Akima1DInterpolator
-from scipy.optimize import minimize_scalar, newton
+from scipy.optimize import minimize_scalar, brentq
```

Comment:
Though normally I ignore, there are a large number of annotations about test coverage here. I seem to have read that this is "still in the works". Guess is Howard ok with this?

Reply:
Re-iterating the concern of apparent lack of test coverage.

Reply:
Agree that we at least need more regtests of this extraction applied with various param settings, etc.